pkg/util/validation/validation.go

```diff
@@ -84,6 +87,18 @@ func ValidateOverrideSpec(overrideSpec *policyv1alpha1.OverrideSpec) error {
 		return fmt.Errorf("overrideRules and (overriders or targetCluster) can't co-exist")
 	}
 
+	for _, rule := range overrideSpec.OverrideRules {
+		for _, annotation := range rule.Overriders.AnnotationsOverrider {
+			if err := ValidateAnnotations(annotation); err != nil {
+				return err
+			}
+		}
+		for _, label := range rule.Overriders.LabelsOverrider {
+			if err := ValidateLabels(label); err != nil {
+				return err
+			}
+		}
+	}
```

Comment:
Can you help add some ut for this new logic?

Reply:
ok